browser/brave_shields/ad_block_subscription_download_manager_getter.cc

```diff
@@ -102,10 +104,11 @@ class AdBlockSubscriptionDownloadManagerGetterImpl
   }
 
   void OnProfileManagerDestroying() override {
-    g_browser_process->profile_manager()->RemoveObserver(this);
+    profile_manager_->RemoveObserver(this);
```

Comment:
this code is called from ProfileManager destructor when `g_browser_process->profile_manager()` is null already. 
cc @antonok-edm